src/gui/mzroll/pollyelmaveninterface.cpp

```diff
@@ -966,6 +1117,107 @@ QStringList PollyElmavenInterfaceDialog::_prepareFilesToUpload(QDir qdir,
     return filenames;
 }
 
+QStringList
+PollyElmavenInterfaceDialog::_prepareSessionFiles(QString datetimestamp)
+{
+    QStringList filenames;
+
+    // write a JSON and CSV file for each peak table
+    QList<QPointer<TableDockWidget>> peakTableList =
+        _mainwindow->getPeakTableList();
+    peakTableList.append(_mainwindow->getBookmarkedPeaks());
+    for (TableDockWidget* peakTable : peakTableList) {
+        if (peakTable->getGroups().isEmpty())
+            continue;
+
+        QString tableName = TableDockWidget::getTitleForId(peakTable->tableId);
```

Comment:
The table name could have spaces, might be a good idea to strip it before using it as the filename.

Reply:
The next line (1134) does exactly this.

Reply:
Sorry for that. Distracted by other stuff.